lib/cequel/record/tasks.rb

```diff
@@ -63,12 +63,15 @@ def drop!
     puts "Dropped keyspace #{Cequel::Record.connection.name}"
   end
 
+  def models_dir_path
+    ENV['CEQUEL_MODELS_PATH'] || "#{File.expand_path('app/models', project_root)}/"
```

Comment:
`ENV.fetch('CEQUEL_MODELS_PATH', project_root + "app/models")`? We'd have to ensure that `project_root` was a `Pathname` but that would be a Good Thing.